workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/fragments/group/liferay-learn/fragments/unauthenticated-top-banner/index.js

```diff
@@ -4,12 +4,17 @@
  */
 
 if (!themeDisplay.isSignedIn()) {
-	document.querySelector('.learn-banner').style.marginTop = '150px';
+	const publicSiteNavigationContainer = document.querySelector(
+		'.public-site-navigation-container'
+	);
+	publicSiteNavigationContainer.style.marginTop = '50px';
```

Comment:
Hi @lluk4z , 
Let's avoid inline style. Instead, you can apply a class and set the attibutes in the css.